osmdroid-android/src/main/java/org/osmdroid/views/Projection.java

```diff
@@ -675,17 +675,36 @@ public void adjustOffsets(final IGeoPoint pGeoPoint, final PointF pPixel) {
 	 * or it is smaller and it is centered
 	 * @since 6.0.0
 	 */
+	@Deprecated
 	public void adjustOffsets(final BoundingBox pBoundingBox) {
 		if (pBoundingBox == null) {
 			return;
 		}
-		final long left = getLongPixelXFromLongitude(pBoundingBox.getLonWest());
-		final long right = getLongPixelXFromLongitude(pBoundingBox.getLonEast());
-		final long top = getLongPixelYFromLatitude(pBoundingBox.getActualNorth());
-		final long bottom = getLongPixelYFromLatitude(pBoundingBox.getActualSouth());
-		final long deltaX = checkScrollableOffset(left, right, mMercatorMapSize, mIntrinsicScreenRectProjection.width());
-		final long deltaY = checkScrollableOffset(top, bottom, mMercatorMapSize, mIntrinsicScreenRectProjection.height());
-		adjustOffsets(-deltaX, -deltaY);
+		adjustOffsets(pBoundingBox.getLonWest(), pBoundingBox.getLonEast(), false, 0);
+		adjustOffsets(pBoundingBox.getActualNorth(), pBoundingBox.getActualSouth(), true, 0);
+	}
+
+	/**
+	 * @since 6.0.0
+	 */
+	public void adjustOffsets(final double pNorthOrWest, final double pSouthOrEast,
```

Comment:
this is public, why would someone want to call this? not clear (to me) based on the method name or the javadocs. What offset is being adjusted?

Reply:
Currently commenting and downgrading access to package-private.